lib/krikri/mapper.rb

```diff
@@ -135,7 +135,7 @@ def run(activity_uri = nil)
             rec.mint_id! if rec.node?
             activity_uri ? rec.save_with_provenance(activity_uri) : rec.save
           rescue => e
-            Rails.logger.error("Error saving record: #{rec.rdf_subject}\n" \
+            Rails.logger.error("Error saving record: #{rec.try(:rdf_subject)}\n" \
```

Comment:
`#try` is a method on `Object`.  It's basically `#send` except it returns `nil` instead of throwing an error if the method is undefined.

http://apidock.com/rails/Object/try